### ISSUE TITLE
Add any of your basic python codes

### ISSUE THREAD

**Issue by SJ-Kumar** [SJ-Kumar/hacktoberfest2022#5]

**Comment by VS-Babu** [comment]
I will add basic API usage pgm in python.

**Comment by mak650650** [comment]
is there any minimum requirement for code to qualified

**Comment by SJ-Kumar** [comment]
> is there any minimum requirement for code to qualified

No, this is just for practice. As per my knowledge, they will look for genuine commits. The first 40,000 will be eligible for goodies.

**Comment by gaurav03kr** [comment]
can you please assign this to me i want to make pr i code it already.thanks.

**Comment by SJ-Kumar** [comment]
If any of you interested in creating html/js pages ...please join the following repository....
https://github.com/SJ-Kumar/studentfolio/issues
I am still working on this repository....I can share the exact details in a day or two.
